.dockerignore

```diff
@@ -14,6 +14,7 @@
 !internal/
 !pkg/
 !scripts/
+!certs/*
```

Comment:
If we're going to pass this into the docker build context, do we want to store the cert in a regular file in the certs directory, so we don't have to `<<EOF` it in the Dockerfile?